### PR TITLE
Fix #207

### LINKED ISSUE #207: Cambrian, Permian etc.
The hyponyms of geological period (`ewn-15272329-n`) are defined starting with the word 'from'

for example

`ewn-15150049-n` (Interlingual Index: i116681)
> Tertiary period, Tertiary - from 63 million to 2 million years ago

It would be better if the phrase "the period" as prepended to make the definitions a little clearer

### PR DESCRIPTION
**Pull request by jmccrae**
Fix issue relating to these definitions

### CODE REVIEW

**Comment by arademaker** [comment]
We have many new definitions of geological periods in our previous paper about it  "Extending Wordnet to Geological Times" GWC 2018. Can we submit a PR with new synsets?

**Comment by jmccrae** [comment]
@arademaker. Yes, feel free to contribute :)

**Comment by arademaker** [comment]
How are you editing the files? Any direction? I don't want to introduce conflicts that could be avoided since you are also changing files to close issues, right?

File: src/wn-noun.time.xml [hunk 1]
```diff
@@ -8079,19 +8079,19 @@
...
     <Synset id="ewn-15151313-n" ili="i116688" partOfSpeech="n" dc:subject="noun.time">
-      <Definition>from 135 million to 63 million years ago; end of the age of reptiles; appearance of modern insects and flowering plants</Definition>
+      <Definition>the period from 135 million to 63 million years ago; end of the age of reptiles; appearance of modern insects and flowering plants</Definition>
       <SynsetRelation relType="instance_hypernym" target="ewn-15272329-n"/> <!-- period, geological period -->
       <SynsetRelation relType="holo_part" target="ewn-15151127-n"/> <!-- Age of Reptiles, Mesozoic era, Mesozoic -->
     </Synset>
     <!-- Jurassic, Jurassic period -->
     <Synset id="ewn-15151547-n" ili="i116689" partOfSpeech="n" dc:subject="noun.time">
-      <Definition>from 190 million to 135 million years ago; dinosaurs; conifers</Definition>
+      <Definition>the period from 190 million to 135 million years ago; distinguished by dinosaurs and confiers</Definition>
```
Comment (alexandretessarollo) [inline src/wn-noun.time.xml]: typos, confers should be conifers

Comment (jmccrae) [reply src/wn-noun.time.xml]: Thanks, fixed!

**Comment by jmccrae** [comment]
Hi @arademaker ,

There are some scripts in `scripts/` that make some changes although they are still WIP. Conflicts should not be a problem as long as you don't do anything like reformat the XML :)

**Pull request merged**